vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/ConfigurationBusiness.java

```diff
@@ -985,15 +986,24 @@ public User getOrCreateUser(String email, String defaultAccountType)
                 }
             }
 
-            user = getNewUser(email, firstName, lastName);
+            if(institution != null){
```

Comment:
Move this block to `EgiController`, in a new method called before `getOrCreateUser`.

This should not be there, because `getOrCreateUser` must receive a good institution in a string (like `CNRS`) but not a email or any strange format.

Reply:
Done